ephios/core/templates/core/serviceworker.js

```diff
@@ -33,6 +33,17 @@ self.addEventListener('activate', event => {
     );
 });
 
+self.addEventListener("message", (event) => {
+    if (event.data === "logout") {
+        event.waitUntil(
+            caches.keys().then(cacheNames => {
+                console.log("Clearing cache after logout");
+                return Promise.all(cacheNames.map(cacheName => caches.delete(cacheName)));
+            })
+        );
+    }
+});
```

Comment:
I did test this by fixing the cache name and checking that the cache is cleared when logging out in chrome and firefox.